agency/README.md

```diff
@@ -365,20 +365,20 @@ The reservation method enum describes the different ways in which a passenger ca
+| dispatch_time         | Timestamp     | Conditionally Required | Time the vehicle was dispatched to the customer (required if trip was dispatched)                                                                                                                                                                                                                                  |
+| trip_start_time       | Timestamp     | Conditionally Required | Time the trip started (required if trip started)                                                                                                                                                                                                                                                                   |
+| trip_end_time         | Timestamp     | Conditionally Required | Time the trip ended (required if trip was completed)                                                                                                                                                                                                                                                               |
+| distance              | Float         | Conditionally Required | Total distance of the trip in meters (required if trip was completed)                                                                                                                                                                                                                                              |
+| cancellation_reason   | string        | Conditionally Required | The reason why a *driver* cancelled a reservation. (required if a driver cancelled a trip, and a `driver_cancellation` event_type was part of the trip)                                                                                                                                                            |
+| fare                  | [Fare](#fare) | Conditionally Required | Fare for the trip (required if trip was completed)                                                                                                                                                                                                                                                                 |
+| accessibility_options | Enum[]        | Optional               | The **union** of any accessibility options requested, and used. E.g. if the passenger requests a vehicle with `wheelchair_boarding`, but doesn’t utilize the features during the trip, the trip payload will include `accessibility_options: ['wheelchair_boarding']`. See [accessibility-options][accessibility-options] |
```

Comment:
Should this be `wheelchair_accessible` instead of `wheelchair_boarding` as the example text?

Reply:
Derp, yes. I keep getting confused with which one we're using, heh.